fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -194,7 +193,7 @@ public void scenario1() throws IOException {
     }
 
     @Test
-    @Ignore("Until WebACAuthorizingRealm is complete: https://jira.duraspace.org/browse/FCREPO-2760")
+    @Ignore("needs principal providers config")
```

Comment:
Is there a JIRA for this? `needs principle providers config`
If so, please add a link.
If not, please create and add the link.